test/E2ETest/testapps/nuget.config

```diff
@@ -3,10 +3,4 @@
   <packageSources>
     <add key="dev" value="..\PackageTest\packages" />
   </packageSources>
-  <packageSourceMapping>
-    <packageSource key="dev">
-      <package pattern="BytecodeAlliance.Componentize.DotNet.WitBindgen" />
-      <package pattern="BytecodeAlliance.Componentize.DotNet.Wasm.SDK" />
-    </packageSource>
-  </packageSourceMapping>
```

Comment:
why removing these?  I think it is needed to redirect nuget to the pacakge source of `value="..\PackageTest\packages"`

Reply:
I had problems with them locally, but will try CI

Reply:
Looks fine with them, maybe it was the Byte[Cc] causing me issues.  Thanks!

Reply:
The diff looks like these are still removed?  Am I reading that correctly?

Reply:
> Looks fine with them, maybe it was the Byte[Cc] causing me issues. Thanks!

this probably was causing issues!

Reply:
Seems like Linux doesn't like the package source mapping after all, I'm not really sure why it is needed.
```
/home/runner/work/componentize-dotnet/componentize-dotnet/test/E2ETest/testapps/E2EConsumer/E2EConsumer.csproj : error NU1100: Unable to resolve 'BytecodeAlliance.Componentize.DotNet.Wasm.SDK (>= 0.2.0-dev)' for 'net8.0'. PackageSourceMapping is enabled, the following source(s) were not considered: /home/runner/work/componentize-dotnet/componentize-dotnet/test/E2ETest/testapps/../PackageTest/packages, dotnet-experimental, dotnet8, nuget. [/home/runner/work/componentize-dotnet/componentize-dotnet/test/E2ETest/PackageTest/PackageTest.csproj]
```

Reply:
My understanding is that it re-directs nuget to the local packages folder (`test/E2ETest/testapps/../PackageTest/packages`) instead of using the default nuget (`~/.nuget/packages`) folders.  This way when building a new pacakge we know it is using the new version built from source and not the cached version.

Maybe something else is going on. I will try this out today on my linux box

Reply:
you beat me to it by 5 mins, I pushed to your branch and was blocked

this is the diff I came up with that worked:

```xml
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="dev" value="..\PackageTest\packages" />
+    <add key="dev" value="../PackageTest/packages" />
   </packageSources>
   <packageSourceMapping>
     <packageSource key="dev">
-      <package pattern="BytecodeAlliance.Componentize.DotNet.WitBindgen" />
-      <package pattern="BytecodeAlliance.Componentize.DotNet.Wasm.SDK" />
+      <package pattern="BytecodeAlliance.Componentize.*" />
+      <package pattern="BytecodeAlliance.Componentize.*" />
     </packageSource>
   </packageSourceMapping>
 </configuration>

```

Reply:
looks like the pattern matching wasn't needed just the forward slashes!